src/opentom_skel/start.sh

```diff
@@ -31,6 +31,10 @@ export NANOX_YRES=`fbset -s | grep geometry | if read x x yres x; then echo $yre
 # Verify TS is calibrated
 if [ ! -f $TSLIB_CALIBFILE ]; then ts_calibrate; fi
 
+# make ". s" usable to get environment in telnet 
```

Comment:
yes, this would be great but it did not work with the minimal telnetd.
I tried something, but found no real solution.